runlike/inspector.py

```diff
@@ -90,6 +95,14 @@ def parse_ports(self):
                         '--expose=%s' %
                         container_port_and_protocol)
 
+        exposed_ports = self.get_fact("Config.ExposedPorts")
+        if exposed_ports:
```

Comment:
```suggestion
        if exposed_ports and not ports:
```

Reply:
I believe "NetworkSettings.Ports" and "Config.ExposedPorts" may both exist.